src/controllers/reactWebviewController.ts

```diff
@@ -5,6 +5,7 @@
 
 import * as vscode from 'vscode';
 import { WebviewRoute } from '../sharedInterfaces/webviewRoutes';
+import { getNonce } from '../reactviews/common/utils';

```

Comment:
Can we keep react views folder for only things used within the react app? I will try to add a linter rule to prevent such import.

Reply:
maybe more common utility functions can go into 'src/utils/utils.ts'?